### PR TITLE
[qt5-base] fix linking of SQLite in vcpkg-cmake-wrapper.cmake

### PR DESCRIPTION
**Pull request by Be-ing**
FindSQLite3.cmake included in CMake exports a SQLite::SQLite3
target:
https://cmake.org/cmake/help/latest/module/FindSQLite3.html

### CODE REVIEW

**Comment by Be-ing** [comment]
I am puzzled how this worked before.

File: overlay/osx/qt5-base/vcpkg-cmake-wrapper.cmake [hunk 1]
```diff
@@ -22,7 +22,7 @@ if("${_target_type}" STREQUAL "STATIC_LIBRARY")
     find_package(harfbuzz CONFIG)
 
     set_property(TARGET Qt5::Core APPEND PROPERTY INTERFACE_LINK_LIBRARIES
-        ZLIB::ZLIB JPEG::JPEG PNG::PNG Freetype::Freetype SQLite3::SQLite3 harfbuzz::harfbuzz
+        ZLIB::ZLIB JPEG::JPEG PNG::PNG Freetype::Freetype SQLite::SQLite3 harfbuzz::harfbuzz
```
Comment (Holzhaus) [inline overlay/osx/qt5-base/vcpkg-cmake-wrapper.cmake:25]: Are you sure this is correct?

In Mixxx `CMakeLists.txt`, we also link to `SQLite3::SQLite3`...

**Review by Holzhaus — APPROVED** [review]

File: overlay/osx/qt5-base/vcpkg-cmake-wrapper.cmake (hunk 1 above)
Comment (Be-ing) [reply overlay/osx/qt5-base/vcpkg-cmake-wrapper.cmake:25]: This is changed in https://github.com/mixxxdj/mixxx/pull/4432

Comment (Holzhaus) [reply overlay/osx/qt5-base/vcpkg-cmake-wrapper.cmake:25]: ok

**Pull request merged**